### PR TITLE
jax.numpy.linalg.tensorinv

### PR DESCRIPTION
**Pull request by 0xcro3dile**

### CODE REVIEW

**Comment by JamieLine** [comment]
Hi! Thanks for the PR!

Before I can merge it, it will need to have complete tests and those test must pass. That means that roughly the following things must be done.

1. The linting errors found by GitHub Actions must be fixed. These will be simple to fix, you can either read off the relevant lines from the log found higher up on this webpage, or you can run `flake8` and `black` locally to find and fix them. You can also set-up `pre-commit` as described in the docs to have these tests run automatically in the future, but unfortunately that will not run them retroactively so the current errors will still be there until they are manually fixed.
2. In my Codespaces environment, the given test for the JAX frontend fails. Run the test locally using `pytest` and the failures should be fairly easy to see. None of the errors I got were too bad to fix, but I don't know if they are hiding other errors. This step shouldn't be too bad but might take some work. If you'd like any help getting `pytest` to run, or you get any errors you can't find a way to fix, feel free to let me know and I'll see if I can help.
3. There needs to be a test for the NumPy frontend as well, and that test needs to pass. Once you've got the JAX test passing, the same test configuration will probably be correct for the NumPy test, and then as long as the NumPy frontend works, that test should pass. If you'd like, you can remove the NumPy frontend implementation from this PR and re-submit it for a new PR if you want to get this one to merge faster.

Overall though, the PR looks good, everything here seems broadly sensible. 

Any questions or comments feel free to let me know!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
